gems/smithy/lib/smithy/views/client/client.rb

```diff
@@ -16,6 +16,10 @@ def module_name
           @plan.module_name
         end
 
+        def service_id
+          @plan.service.keys.first
+        end
```

Comment:
Note to self: I think it is supposed to be the full service-id namespace. I need to check docs / code.

Reply:
Yeah - I think it is.

Reply:
I think I won't need this - if we are relying on the schema to tell us the service id shape (should we need it).